group-actors/src/main/resources/activityConfig.json

```diff
@@ -0,0 +1,23 @@
+{
+  "activities":[
+    {
```

Comment:
@sknirmalkar89 - the structure of activityConfig is different from what was shared yesterday. Why did you do these changes?